### PR TITLE
Improved operation name logging for Netty based applications

### PR DESCRIPTION
**Pull request by neinoi**
Fix Operation name logging for Netty based applications

No discussion found on this subject, nor issue.

### CODE REVIEW

File: instrumentation/netty/netty-3.8/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/netty/v3_8/server/HttpServerRequestTracingHandler.java [hunk 1]
```diff
@@ -46,7 +46,7 @@ public void messageReceived(ChannelHandlerContext ctx, MessageEvent event) {
     HttpRequest request = (HttpRequest) message;
 
     Context context =
-        tracer().startSpan(request, ctx.getChannel(), channelTraceContext, "netty.request");
+        tracer().startSpan(request, ctx.getChannel(), channelTraceContext, request.getMethod() + " " + request.getUri());
```
Comment (trask) [inline instrumentation/netty/netty-3.8/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/netty/v3_8/server/HttpServerRequestTracingHandler.java:49]: we need to follow OpenTelemetry semantic conventions (see https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/semantic_conventions/http.md#name)

```suggestion
        tracer().startSpan(request, ctx.getChannel(), channelTraceContext, "HTTP " + request.getMethod());
```

if you want a more specific operation name including the URI, check out https://docs.microsoft.com/en-us/azure/azure-monitor/app/java-standalone-upgrade-from-2x#operation-names

**Review by trask — COMMENTED** [review]
Hi @neinoi! Can you submit this PR upstream to https://github.com/open-telemetry/opentelemetry-java-instrumentation? Then we will pull it in to our fork after the next upstream release.

File: instrumentation/netty/netty-3.8/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/netty/v3_8/server/HttpServerRequestTracingHandler.java (hunk 1 above)
Comment (neinoi) [reply instrumentation/netty/netty-3.8/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/netty/v3_8/server/HttpServerRequestTracingHandler.java:49]: Hi @trask , I need to have more details in my insights. With my proposition I get something like this in the operation name : 
GET /api/v3/tracks

Before that, all calls were grouped by « netty.request » name.
With your proposition, I’ll have something like that : 
HTTP GET
HTTP POST

(For the upstream PR, no pb)

**Comment by trask** [comment]
closing, this was merged upstream https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/2317, thx @neinoi!

**Pull request closed without merging**